dev/tools/update_icons.dart

```diff
@@ -349,14 +408,15 @@ class _Icon {
       htmlSuffix = '-filled';
     } else {
       family = 'material';
-      if (id.endsWith('_outlined') && id != 'insert_chart_outlined') {
```

Comment:
Is checking for `insert_chart_outlined` no longer necessary?

Reply:
Exactly